### ISSUE TITLE
Changes of extensions in the "info" section of OpenAPI documents are not detected

### ISSUE THREAD

**Issue by sebkunz** [pb33f/libopenapi#184]
Hi,

I am currently trying out **libopenapi** to detect changes in OpenAPI documents and recognized that changes of extensions in the `info` section of the OpenAPI are not detected.

It seems like extensions are generally supported and checked for changes in other sections of the OpenAPI document (e.g. in `operation paths` or `components`).

Is there a specific reason why the `info` section is not supported or is it a bug?

Would love to see detection of changes for extensions in the `info` section as well since I do have some extensions set in the `info` sections of my OpenAPI documents.

**Example**:

Having two OpenAPI documents with an extension change in the info section from
```yaml
# original doc
info:
  title: Sample API
  version: 1.0.0
  x-extension: info section extension
``` 
to
```yaml
# updated doc
info:
  title: Sample API
  version: 1.0.0
  x-extension: info section extension of the second document
``` 
and detecting changes via 

```go
// libopenapi version v0.12.1
documentChanges, errs := libopenapi.CompareDocuments(originalDoc, updatedDoc)
```
does not detect the change in the extension and returns no changes (and no errors).

Hope this minimal example helps you to reproduce my issue.
Thanks in advance!

**Comment by daveshanley** [comment]
You have discovered a small gap!

https://github.com/pb33f/libopenapi/blob/main/what-changed/model/info.go#L142

At the bottom, the 'extensions' check is being missed. The code looks like this:

https://github.com/pb33f/libopenapi/blob/main/what-changed/model/media_type.go#L146

Please feel free to add this code in (and a test) and submit a PR! Join the community!

**Comment by sebkunz** [comment]
@daveshanley thanks for the fast reply! I added a PR to fix the missing change detection for the info section.

**Comment by daveshanley** [comment]
Fixed in `v0.13.1` Thank you for your contribution!
